rewards/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.13.4 as builder
```

Comment:
Do you think it would be a good idea to have a parent docker file all the sub go-services could consume?

Reply:
Not sure... I think we can eventually use the same dockerfile for all sub services, we have two right now because grant is built completely differently than this new rewards service.